### PR TITLE
step description updates

### PR DESCRIPTION
**Pull request by BanyikAnna**

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -1,6 +1,29 @@
-title: Android Lint
-summary: This step runs lint gradle task on your project.
-description: This step runs lint gradle task on your project.
+title: "Android Lint"
+summary: |-
+  Runs Lint on your Android build's code and detects potential syntax errors to keep your code error free.
```
Comment (godrei) [inline step.yml]: Android build's code -> Android project source files

File: step.yml [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+description: |-
+  It highlights the code line where the error is found, explains the type of error and suggests correction. This Step does not make the build fail if it spots any structural errors in your code. The results are available in a viewable or downloadable Lint Report html file which you can access on the Build's APPS & ARTIFACTS page.
```
Comment (godrei) [inline step.yml]: Lint Report html file -> Lint Report HTML or XML file

(see: report_path_pattern/summary)

Comment (godrei) [reply step.yml]: which you can access on the Build's APPS & ARTIFACTS page. -> should not we mention that this needs to have the Deploy to Bitrise.io step in the workflow? (without that step it is not true)

File: step.yml [hunk 3]
```diff
@@ -1,6 +1,29 @@
...
+
+  ### Configuring the Step
+
+  1. Set the **Project Location** input which, by default, points to the root directory of your Android project.
+  2. Set the module and variant you wish to lint in the **Module** and **Variant** fields.
+  Optionally, you can modify these inputs:
+  1. You can set where the lint report should be exported to if you overwrite the **Report location pattern** input. By default, it's set to `*/build/reports/lint-results*.html`.
```
Comment (godrei) [inline step.yml]: You can set where the lint report should be exported to -> You can specify where the lint report can be found in your project after running Android Lint.

File: step.yml [hunk 4]
```diff
@@ -1,6 +1,29 @@
...
+  2. You can set if the Step should cache the build cache with dependencies, only the dependencies or nothing at all in the **Set level of cache** input.
```
Comment (godrei) [inline step.yml]: You can set if the Step should cache build outputs and dependencies, only the dependencies or nothing at all in the **Set level of cache** input.

File: step.yml [hunk 5]
```diff
@@ -1,6 +1,29 @@
...
+  3. You can set any gradle argument to a gradle task in the **Additional Gradle Arguments** input.
```
Comment (godrei) [inline step.yml]: to the gradle task

**Review by godrei — CHANGES_REQUESTED** [review]
Pls check my notes

File: step.yml (hunk 2 above)
Comment (BanyikAnna) [reply step.yml]: will do :)

**Review by godrei — APPROVED** [review]

**Pull request merged**